src/migrations/1724333243087-addBoKLastUpdateToAiPersonaService.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddBoKLastUpdateToAiPersonaService1724333243087
+  implements MigrationInterface
+{
+  name = 'AddBoKLastUpdateToAiPersonaService1724333243087';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE \`ai_persona_service\` ADD \`bodyOfKnowledgeLastUpdated\` datetime DEFAULT NULL`
+    );
+
+    await queryRunner.query(
+      `UPDATE \`ai_persona_service\` SET \`bodyOfKnowledgeLastUpdated\`= NOW()`
```

Comment:
I don't think this is a good idea. It will make every vc `ready`.
Maybe a better approach is to conditionally set it if a body of knowledge is defined.

Reply:
Having a VC without a BoK would be a data inconsistency error. I've added that for backward compatibility - if you have a VC you have used all of a sudden, it'd pop up as initializing on the FE.

Reply:
it's because BoK is nullable in the database but probably enforced with other means.